File: Resueltos_por_competidor/Miguel_Lopez/RPC-04/Cheater.py
n = int(input())
for _ in range(n):
    s = input()

    a = s.split(".")
    b = int(a[0])
    c = int(a[1])

    d = b * 100 + c

    e = d % 3

    if e == 0 or e == 1:
        print("VALID")
    else:
        print("IMPOSSIBLE")

File: Resueltos_por_competidor/Jacobo_Segura/Cheater.java
import java.util.Scanner;

public class Cheater {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        
        for(int i = 0; i<n; i++) {
        	String s = sc.next();
        	int valor = Integer.parseInt(s.replace(".", ""));
        	if(valor%3 == 2) {
        		System.out.println("IMPOSSIBLE");
        	}else {
        		System.out.println("VALID");
        	}
        }
    }
}
